### ISSUE TITLE
Discworld Noir crashes to Windows after intro

### ISSUE THREAD

**Issue by Paolo86cripple** [joncampbell123/dosbox-x#2929]
Discworld Noir crashes after intro. Using dynamic_x86 core with software voodoo emulation in a win98 guest.

**Comment by Siddhant-Mashalkar** [comment]
ok, but i have few questions:
1.What error are you getting , like "this program has performed illegal operation"
2.what is config file you are using?
3.what host os are you running, which version of dosbox-x you are running , and is it SDL1 or SDL2
4.log file

try setting "Core=normal" or change cpu emulation to highest

**Comment by Paolo86cripple** [comment]
Sorry for the being unclear.
1. I'm simply getting a simple return to windows without any error messages but with the start menu which is not visible anymore. 
2. I'll attach the config file

[sdl]
autolock=true

[dosbox]
title=Windows 98
memsize=64

[video]
vmemsize=8
vesa modelist width limit=0
vesa modelist height limit=0

[dos]
ver=7.1
hard drive data rate limit=0

[cpu]
cputype=pentium_mmx
core=dynamic_x86

[sblaster]
sbtype=sb16vibra

[fdc, primary]
int13fakev86io=true

[ide, primary]
int13fakeio=true
int13fakev86io=true

[ide, secondary]
int13fakeio=true
int13fakev86io=true
cd-rom insertion delay=4000

[render]
scaler=none

[voodoo]
voodoo_card=auto
voodoo_maxmem=true

[autoexec]
IMGMOUNT C hdd.img
IMGMOUNT D dnoir101.iso
BOOT C:

3. My host OS is Manjaro Linux, running SDL2 0.83.17,
4. Where are the log files located? .config and the working directory have no log files.

Already tried setting core=normal, the result is the same. Cpu emulation is already the highest available, which I set manually every time in the menu, 'cause the game already needs a Pentium 166 MMX to run.

**Comment by brunocastello** [comment]
does the game run when you set sbtype=none?

**Comment by Paolo86cripple** [comment]
Yes, it DOES run  perfectly after the intro with sbtype=none for some reason. Don't know why.

**Comment by brunocastello** [comment]
@Paolo86cripple that's what I expected, I have the same issue with a different environment (macOS, SDL1) since June/July. It could be an issue/bug with the SB16 emulation. I am yet to try out other cards like sb pro 1, 2 to check for a more complete understanding of the problem and give the devs a better feedback of the problem. If you could try the other cards too, that would help us both.

@Wengier @rderooy apparently it affects all platforms, not just macOS (I tested Windows under Windows 10 Pro and had the same).

**Comment by Paolo86cripple** [comment]
@brunocastello  sbpro1 and 2 both result in a HARD crash. dosbox does not even return to windows and reboot from the menu is needed. sb16 produces the same results as vibra variant.
